service/models.py

```diff
@@ -147,3 +147,14 @@ def find_by_recommend_type(cls, recommend_type):
         """
         cls.logger.info('Processing recommend_type query for %s ...', recommend_type)
         return cls.query.filter(cls.recommend_type == recommend_type)
+
+    @classmethod
+    def find_by_specifc_info(cls, customer_id, product_id, recommend_type):
```

Comment:
There is already find _by_product_id, find_by_customer_id and find_by_recommend_type in model.py, do you still need find_by_specific_info method?